au.edu.uq.esys.escript.appdata.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file is not used, it should actually be removed. so is the .desktop (since they are installed from upstream source)